tests/foreman/maintain/test_packages.py

```diff
@@ -0,0 +1,293 @@
+
+    result = sat_maintain.cli.Packages.status()
+    assert result.status == 0
+    assert 'FAIL' not in result.stdout
+    assert 'Packages are not locked.' in result.stdout
+    assert 'Automatic locking of package versions is disabled in installer.' in result.stdout
+
+    result = sat_maintain.cli.Packages.is_locked()
+    assert 'Packages are not locked' in result.stdout
+    assert result.status == 1
+
+    result = sat_maintain.execute('dnf remove -y zsh')
+    assert result.status == 0
+
+    result = sat_maintain.execute('dnf install -y zsh')
+    assert result.status == 0
+    assert 'Use foreman-maintain packages install/update <package>' not in result.stdout
+
+    @request.addfinalizer
+    def _finalize():
+        assert sat_maintain.execute('dnf remove -y zsh').status == 0
+        if sat_maintain.__class__.__name__ == 'Satellite':
+            result = sat_maintain.install(InstallerCommand('lock-package-versions'))
+            assert result.status == 0
+            assert 'Success!' in result.stdout
+
+
+@pytest.mark.include_capsule
+def test_positive_fm_packages_update(request, sat_maintain):
+    """Verify whether packages check-update and update work as expected.
+
+    :id: 354d9940-10f1-4244-9079-fdbd24be49b3
+
+    :parametrized: yes
+
+    :steps:
+        1. Run satellite-maintain packages check-update
+        2. Run satellite-maintain packages update to update the walrus package.
+
+    :expectedresults: check-update subcommand should list walrus package for update
+        and update subcommand should update the walrus package.
+
+    :BZ: 1803975
+
+    :customerscenario: true
+    """
+    # Setup custom_repo and packages update
+    sat_maintain.create_custom_repos(custom_repo=settings.repos.yum_0.url)
+    result = sat_maintain.cli.Packages.install(packages='walrus', options={'assumeyes': True})
+    assert result.status == 0
+    assert (
+        sat_maintain.execute(
+            'dnf -y --disableplugin=foreman-protector downgrade walrus-0.71-1'
+        ).status
+        == 0
+    )
+
+    # Run satellite-maintain packages check update and verify walrus package is available for update
+    result = sat_maintain.cli.Packages.check_update()
+    assert 'FAIL' not in result.stdout
+    assert result.status == 0
+    assert 'walrus' in result.stdout
+    # Run satellite-maintain packages update
+    result = sat_maintain.cli.Packages.update(packages='walrus', options={'assumeyes': True})
+    assert 'FAIL' not in result.stdout
+    assert result.status == 0
+    # Verify walrus package is updated.
+    result = sat_maintain.execute('rpm -qa walrus')
+    assert result.status == 0
+    assert 'walrus-5.21-1' in result.stdout
+
+    @request.addfinalizer
+    def _finalize():
+        assert sat_maintain.execute('dnf remove -y walrus').status == 0
+        sat_maintain.execute('rm -rf /etc/yum.repos.d/custom_repo.repo')
+
+
+@pytest.mark.stubbed
+def test_positive_fm_packages_sat_installer(sat_maintain):
```

Comment:
Add :testAutomation: token for not automated.

Reply:
@jyejare Updated, request to re-review

Reply:
Is this Manual-Only or :NotAutomated:

There is a difference in both of these tokens:

- ManualOnly = That never suppose to Automate and will always be run manually
- NotAutomsted = Not automated currently but should be automated in future.

![Screenshot 2022-11-16 at 5 40 15 PM](https://user-images.githubusercontent.com/11752425/202177349-66bb0184-c6a9-41f8-99f8-f85009e00d1c.png)

Reply:
@jyejare Yes, I know, ManualOnly is correct here